modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppUE4ClientCodegen.java

```diff
@@ -497,13 +497,20 @@ public String toVarName(String name) {
             name = name.toLowerCase(Locale.ROOT);
         }
 
+        //Unreal variable names are CamelCase
+        String camelCaseName = camelize(name, false);
+
+        //Avoid empty variable name at all costs
+        if(!camelCaseName.isEmpty()) {
+            name = camelCaseName;
+        }
+
         // for reserved word or word starting with number, append _
         if (isReservedWord(name) || name.matches("^\\d.*")) {
             name = escapeReservedWord(name);
         }
```

Comment:
I think the special handling of the reserved words may not work here (before and after this PR).

I'll file another PR to address it later.